cura/Machines/Models/MachineListModel.py

```diff
@@ -50,6 +54,18 @@ def __init__(self, parent=None) -> None:
         CuraContainerRegistry.getInstance().containerRemoved.connect(self._onContainerChanged)
         self._updateDelayed()
 
+    showCloudPrintersChanged = pyqtSignal(bool)
+
+    @pyqtProperty(bool, notify=showCloudPrintersChanged)
+    def showCloudPrinters(self) -> bool:
+        return self._show_cloud_printers
+
+    @pyqtSlot(bool, name="setShowCloudPrinters")
+    def set_show_cloud_printers(self, show_cloud_printers: bool) -> None:
```

Comment:
This is a function, and it's used as a function, so should be camelCased. (Although it looks deliberate; where there any problems with the 'normal' way of doing things?)